README.md

```diff
@@ -87,6 +87,7 @@
 - [Base 2](http://menial.co.uk/base/) - A GUI for managing SQLite databases.
 - [Cakebrew](https://www.cakebrew.com/) - The Homebrew GUI App. [![Open-Source Software][OSS Icon]](https://github.com/brunophilipe/Cakebrew)
 - [Charles](http://www.charlesproxy.com) - Charles is a Proxy that allow you to view all of the HTTP and HTTPS traffic.
+- [CocoaRestClient](https://github.com/mmattozzi/cocoa-rest-client) - An app for testing REST endpoints. [![Open-Source Software][OSS Icon]](https://github.com/nlf/dlite)
```

Comment:
The primary link should point to https://mmattozzi.github.io/cocoa-rest-client/ and
The OSS icon should point to the Github Source.

Fix this and I'lll merge :)